stats.lua

```diff
@@ -392,6 +402,54 @@ local function append_display_sync(s)
 end
 
 
+local function append_filters(s, prop, prefix)
+    local length = 0
+    local filters = {}
+
+    local speed = mp.get_property_osd("speed")
+    if speed ~= "1.00" then
+        length = length + string.len("speed []") + speed:len()
+        filters[#filters+1] = "speed" .. it(" [" .. speed .. "]")
+    end
+
+    if prop == "vf" and mp.get_property_bool("deinterlace") then
```

Comment:
> so it should always reflect on whether deinterlacing

The property `deinterlacing` not a filter, it is just an echo of an option a user set. The actual deinterlacing filter that _might get inserted_ is e.g. `yadif` and it will be (should be) correctly listed among the other real video filters.
There's also no point in listing "deinterlacing" and then "yadif" as a filter anyway.